libraries/core-react/src/Divider/Divider.tsx

```diff
@@ -17,25 +17,25 @@ const StyledDivider = styled.hr<StyleProps>`
   height: ${(props) => props.dividerHeight};
 `
 
-type Props = {
-  /** Color variants
- @default medium */
+export type DividerProps = {
```

Comment:
Is there a specific reason you renamed from `Props` to `DividerProps`? We have been using `Props` naming on all components for the "main" component properties.

Reply:
We have to export the types to use them in the stories. I forgot that yesterday, so it wasn’t that clear, but take a look at [the story](https://github.com/equinor/design-system/blob/020ce6cacb4366333843d787a1256ef352fc2526/apps/storybook-react/stories/Divider.stories.tsx#L32) where the props are being used, and also index.ts where [the props are exported](https://github.com/equinor/design-system/blob/020ce6cacb4366333843d787a1256ef352fc2526/libraries/core-react/src/index.ts#L5). If we have several components in the same story it’s better to have individual names. Exporting the props like that is removed from the bundle, but something we need to do when we use the typescript files directly in the Storybook.

Reply:
So either we call the explicitly, which means we need to change all existing components `Props` or we can just rename it when importing in the stories. Which do we favour? 🤔

Reply:
I can live with both approaches 🤷‍♀️  Initially I thought that it will be OK to rename imports as most of the components (?) only have one single component, but when I browsed the mighty Internet to see what other people do they often write <Component>Props, including the Storybook docs. So it might be a defacto standard?

Reply:
Yes, I’ve seen that too…